lib/internal/freeze_intrinsics.js

```diff
@@ -20,9 +20,12 @@
 // https://github.com/tc39/proposal-ses/blob/e5271cc42a257a05dcae2fd94713ed2f46c08620/shim/src/freeze.js
 
 /* global WebAssembly, SharedArrayBuffer, console */
-/* eslint-disable no-restricted-globals */
```

Comment:
Does this mean that the new rule doesn't flag all the uses of global objects in this file?

Reply:
@targos I'm sorry, I couldn't get what you mean. Since the rule `no-restricted-globals` was deleted in this PR, we don't need this line.

Reply:
Ah, I see. Let me check my rule is exactly working.

Reply:
@targos It took me a while, but I improved the analysis of the scope. I've fixed lots of lint errors in this file.